src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -153,4 +154,10 @@ public void parseCommand_priorityMissingLevel_throwsParseException() {
                                 PriorityCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased()));
     }
 
+    @Test
+    public void parseCommand_listIncomplete() throws Exception {
+        assertTrue(parser.parseCommand(ListIncompleteCommand.COMMAND_WORD) instanceof ListIncompleteCommand);
+        assertTrue(parser.parseCommand(ListIncompleteCommand.COMMAND_WORD + " 3") instanceof ListIncompleteCommand);
```

Comment:
Could leave some comments further explaining what these testcases does for new users.